sql/modules/Date_Formats.sql

```diff
@@ -0,0 +1,506 @@
+CREATE TABLE date_formats (

```

Comment:
This is a DDL statement which belongs in the sql/changes/ folder, because otherwise it will be executed every time modules are being reloaded. (Hmmm, come to think of that, we probably need tests for it.)